jwst/lib/suffix.py

```diff
@@ -297,8 +297,8 @@ def find_suffixes():
                 step = Step.from_config_file(
                     path.join(config_path, config_file)
                 )
-            except Exception:
-                pass
+            except Exception as err:
```

Comment:
Can we be more specific with the exception type here?  What exceptions does `from_config_file` raise?

Reply:
Well, logically, this routine doesn't particularly care what the error is, simply that, for whatever reason, a step cannot be "bare instantiated".

That being said, and see more about the logging comment below, I am going to be filing a couple of issues that can do away with I believe most of the errors. So far, most of the issues are with defunct/missing/mis-spelled configs.

Reply:
Yeah, that makes sense.  That's exactly my concern.  By just blanket handling all exceptions, things get swept under the rug.  We want it to fail for reasons that we don't expect (missing or misspelled configs, for instance).  But another PR is good.  👍